documentation/book/proc-using-openshift-builds-create-image.adoc

```diff
@@ -7,14 +7,13 @@
 = Using {OpenShiftName} builds and S2I to create new images
 
 {OpenShiftName} supports link:https://docs.openshift.org/3.9/dev_guide/builds/index.html[builds^], which can be used together with the link:https://docs.openshift.org/3.9/creating_images/s2i.html#creating-images-s2i[Source-to-Image (S2I)^] framework to create new container images.
-An {OpenShiftName} build takes a builder image with S2I support together with source code and binaries provided by the user and uses them to build a new container image.
+An {OpenShiftName} build takes a builder image with S2I support together with the connector plugin binaries provided by the user and uses them to build a new container image.
 The newly created container image is stored in {OpenShiftName}'s local container image repository and can be used in deployments.
+
 {ProductName} provides a Kafka Connect builder image, which can be found on {DockerRepository} as `{DockerKafkaConnectS2I}` with this S2I support.
-It takes user-provided binaries (with plugins and connectors) and creates a new Kafka Connect image.
+The S2I image takes user-provided binaries (with plugins and connectors), places them in the `/tmp/kafka-plugins` directory, and creates a new Kafka Connect image with them.
 This enhanced Kafka Connect image can be used with the Kafka Connect deployment.
-
-The S2I deployment provided as an {OpenShiftName} template. It can be deployed from the template using the command-line
-or the {OpenShiftName} console.
+When Kafka Connect is started using the new image, it will automatically load all plugins or connectors from the `/tmp/kafka-plugins` directory.
```

Comment:
From what I saw, it should be this:

```suggestion
When Kafka Connect is started using the new image, it will automatically load all plugins or connectors from the `/tmp/s2i/kafka-plugins` directory.
```

Reply:
Right ... its actually `/tmp/kafka-plugins/s2i` according tot he [source codes](https://github.com/strimzi/strimzi-kafka-operator/blob/4218047e5b2b15a4d2fdabc50d967d6f4cb816cf/docker-images/kafka-connect/s2i/scripts/assemble#L22). Thanks for noticing. I will fix it.